app/controllers/application_controller.rb

```diff
@@ -56,6 +59,10 @@ def admin_session_timed_out?
     admin_signed_in? && session[:last_seen_at] < ADMIN_TIMEOUT_LENGTH_IN_MINUTES.minutes.ago
   end
 
+  def redirect_www
+    redirect_to "https://" + request.host.delete_prefix("www.") + request.fullpath, status: 301 if request.host.start_with("www.")
```

Comment:
`start_with` is predicate method

```suggestion
    redirect_to "https://" + request.host.delete_prefix("www.") + request.fullpath, status: 301 if request.host.start_with?("www.")
```